projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -543,9 +543,9 @@ def close_now(self):
 
             log.info(
```

Comment:
Can you update the description of the PR with a snippet how the message looks like after the change?